tuple/include/array_tuple_sketch.hpp

```diff
@@ -116,6 +111,10 @@ template<typename Array, typename Allocator> class compact_array_tuple_sketch;
 
 /**
  * Update array tuple sketch.
+ * This is a wrapper around tuple sketch to match the functionality and serialization format of ArrayOfDoublesSketch in Java.
+ * For this the sketch must be configured with array<double> or std::vector<double>.
+ * This is a more generic implementation for any integral type (serialization assumes contiguous array size_of(T) * num_values).
```

Comment:
Numeric type, not integral

Reply:
right, integral does not include floating point. arithmetic, perhaps